### ISSUE TITLE
Error: connect ETIMEDOUT

### ISSUE THREAD

**Issue by jkob** [schroffl/teamspeak-query#1]
Code:

		const config = require("./config.js"),
				TeamSpeakQuery = require("teamspeak-query"),
				query = new TeamSpeakQuery(config.server.ip, config.server.port)

		query.send("login", config.server.username, config.server.password)
			.then(() => query.send("use", 1))
			.then(() => query.send("servernotifyregister", {"event":"server"}))
			.then(() => console.log("Done"))
			.catch(err => console.error(`Err: ${err}`))
		query.on("cliententerview", data => console.log(`${data.client_nickname} connected to server.`))

Config.js has the following types:
ip: string,
port: number,
username: string,
password: string

I've tried with another TS server aswell, getting the same error. I know this isn't specific to your package but I see you have a lot of repos regarding TS and node. So you're obviously knowledgable regarding this issue. I coulnd't find any other way to find you. Mind helping me?

**Comment by schroffl** [comment]
Are you using a plain IP address to connect to the server and are you in the same network?
Because it looks like the connection can't even be established.
Could you maybe send your `config.ip`? If you don't want anybody to see, I can delete it as soon as I saw it ;)

**Comment by jkob** [comment]
Yes plain x.x.x.x ip. Tried both ways

Yeah I would have no problem showing the config to you, but it's not my information and I don't want it out in the open. If you add me somewhere I can share it with you in private.

**Comment by schroffl** [comment]
Are you able to connect to the server with the Teamspeak Client?

**Comment by jkob** [comment]
Yes I am able to connect to the server on 2 diffrent IPs. One of which is showing a DNS problem in the TS logs, which it then resolves to the 2nd ip. However, I have been trying to connect to the server directly with the second IP, all this time. Im unsure of the issue here.

**Comment by schroffl** [comment]
You could try to connect via running `telnet 127.0.0.1 10011` in the command prompt (should work on windows, mac and linux) and of course insert your desired ip and port (10011 is the default port).
If that works the issue probably lies within the package.

**Comment by jkob** [comment]
Telnet connection failed to the host, so I guess this whole issue is regarding how to connect to the TS server since I dont have the correct ip/port. Maybe once I can figure out this personal issue, your package will work. You can close this issue since the issue is on my part, not yours.

**Comment by schroffl** [comment]
But if nothing is listening on the port node should throw a `ENOENT` and not a `ETIMEDOUT`. And I'd be glad to help anyways :)

**Comment by jkob** [comment]
Since I have no other means of contacting you, this issue is gonna get spammed. But a little update:

I managed to figure out the port (i think). I connected to my friends server with the default TCP port (10011) and I got into the server. I logged in with the ServerQuery login information and it worked. But now I need to specificy what SID i want to use (I believe). Is there any way to list all virtual servers from within the TELNET client?
